components/cms-controller-manager/pkg/apis/cms/v1alpha1/docstopic_common.go

```diff
@@ -0,0 +1,43 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type CommonDocsTopicSpec struct {
+	DisplayName string `json:"displayName,omitempty"`
+	Description string `json:"description,omitempty"`
+	// +kubebuilder:validation:MinItems=1
+	Sources map[string]Source `json:"sources"`
+}
+
+type DocsTopicMode string
+
+const (
+	DocsTopicSingle  DocsTopicMode = "single"
+	DocsTopicPackage DocsTopicMode = "package"
+	DocsTopicIndex   DocsTopicMode = "index"
+)
+
+type Source struct {
+	Url string `json:"url"`
```

Comment:
Please rename `Url` to `URL`

Reply:
Ok, it needs to be adjusted also in Asset.